### ISSUE TITLE
rm -rf is not a valid command on Windows

### ISSUE THREAD

**Issue by AWulkan** [hejianxian/vddl#12]
I tried to run `npm install`, but I got an error when it got to the `rm -rf dist/` part of the package.json file.
https://github.com/hejianxian/vddl/blob/5ffb105ae49ba1fce59b9dd49161c7050d56bcc4/package.json#L11

One solution would be to include 'shx' and just prepend it infront of non-windows commands. Like they've done in this commit: https://github.com/ReactiveX/rxjs/pull/1681/files

I could submit a PR with it if you want. 

I was planning on trying to fix the duplicate item bug as well (#4).

**Comment by hejianxian** [comment]
Of course, look forward to your PR!

**Comment by hejianxian** [comment]
@AWulkan  For bug #4 , do you have any ideas? I tried some methods, but none of them succeeded.

**Comment by AWulkan** [comment]
I will have to take a look at the code this weekend, probably sunday. It is crucial that it works for me since I'm using it for a real world application.

Before moving to this tool I had built a custom list of draggable items in normal JS, but it lacked nested groups, which later became a requirement by the client. So we moved to Vue and found this neat project. :)

Gonna close this issue and continue in #4.
